src/Utilities/StaticCache.hpp

```diff
@@ -86,45 +50,116 @@ class StaticCacheImpl<T> {
 ///
```

Comment:
I missed this before, sorry, but I think the dox for `StaticCache` should describe and have an example for the enum and mixed enum-range caching.